tools/misc/download_dataset.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import os
+import sys
+from itertools import repeat
+from multiprocessing.pool import ThreadPool
+from pathlib import Path
```

Comment:
we usually use os.path rather than pathlib

Reply:
In my view, the author want to call the function path.mkdir(parents=True, exist_ok=True) to make a directory more securely, which may be convenient to download files in multiple threads.